packaging_automation/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Nit, Consider introducing a `.python-version` file with the intended python version that should be used here.

Reply:
https://github.com/citusdata/tools/issues/234